binder/Passes/CheckDeclarationsPass.cs

```diff
@@ -16,6 +16,11 @@ static string GetBaseName(Declaration decl)
             // Remove get_ or set_ from names in case they are properties.
             var name = decl.Name.TrimStart("get_").TrimStart("set_");
 
+            // Do the same for method names starting with get or set since they
+            // might conflict with properties.
+            name = decl.Name.TrimStart("get").TrimStart("set").
+                             TrimStart("Get").TrimStart("Set");
+
```

Comment:
Does this work with "GetGetFoo()" ? For example, PropertyInfo.GetGetMethod() and GetSetMethod() https://msdn.microsoft.com/en-us/library/system.reflection.propertyinfo.getgetmethod%28v=vs.110%29.aspx?f=255&MSPPError=-2147217396

Reply:
I will check.